Changelogs/1.4.0

```diff
@@ -0,0 +1,21 @@
+1.4.0 Release notes (2021-05-19)
+=============================================================
+
+### Features
+
+* Added support for backend configuration (based on geo-ip) for 'gdprApplies' (IABTCF_gdprApplies) flag. This flag can no longer be passed as parameter to the SDK initializer.
+* Interaction with module on app start is now always asynchronous (where previously during first app launch it was synchronous).
+SDK will always call one of two callback methods from interface 'RingPublishingGDPRShowConsentScreenListener' after set this listener:
```

Comment:
after this listener has been set ? albo w ogóle to potrzebne to zdanie? bo listener chyba powinien być wymagany by zaimplementować

Reply:
właśnie.. czy listener nie powinny być wymagane przez konstruktor?

Reply:
Widzę, że Adam już to potem napisałeś :)

Reply:
Usunąłem to zdanie skoro jest mało zrozumiałe. 

Listenerów nie przekazuje się do konstruktorów. 
Obiekty powinny móc dać się zawsze zbudować/skonstruować bez od razu podpinania do jakiegoś słuchacza na ich stan.
Nasłuch na stan obiektu podpina się i odpina ustawiając listnery. 

W RingPublishingGDPR.initialize() ustawiamy tylko potrzebne parametry.

Potem dla głównego obiektu Application ustawiamy listnener, który nasłuchuje, czy stan zgód się zmienił, jeśli użytkownik chce takich zmian tam nasłuchiwać by zainicjalizować biblioteki. 

Następnie w SplashActicity ustawiamy zamiast wcześniejszejszego synchronicznego zapytania, listner, który czeka na asynchroniczną odpowiedź, czy plansz na zgody ma się pokazać, czy są aktualne i można przejść dalej.

Reply:
tak samo, jak w iOS delegatów nie przekazuje się do konstruktorów. Ale... tu sytuacja jest trochę specyficzna, bo tutaj kod biblioteki powinien wymuszać na developerze użycie naszego listenera, by dobrze zaimplementować jej użycie. Stawiać ścisły wymóg. Metoda jest zawsze opcjonalnym sposobem i już raz się przynajmniej zdarzyło, że np. w Jak Dojadę nie skorzystali z którychś z naszych listenerów z uwagi na ich opcjonalność, a potem zdziwienie że coś nie działa.

Reply:
To przeniosę do initialize() ten aplikacyjny listener. Ten drugi setuje się dopiero w Activity, jak zmienię na shouldShowConsentForm(i tu listener) to jak ktoś chce sprawdzić czy shouldShowConsentForm to musi podać listener.

Reply:
A jednak nie :) Bo jak ktoś ten z onConsentsUpdated() chciałby sobie podpiąć w jakimś innym miejscu aplikacji to już nie może.
A tak może wszędzie gdzie się podoba dodać kolejny listener.